### PR TITLE
Use master build of ClickHouse and update expected errors

### PR DESCRIPTION
**Pull request by qoega**
Continue https://github.com/sqlancer/sqlancer/pull/687

### CODE REVIEW

**Comment by qoega** [comment]
Finally green. CI used latest stable ClickHouse build and it did not include latest fixes. Changed to master.

**Comment by mrigger** [comment]
After updating, some new time-out error appeared:
```
Caused by: java.sql.SQLException: Read timed out, server ClickHouseNode(addr=http:localhost:8123, db=T12_1TLPDistinct)@1979597473
	at com.clickhouse.jdbc.SqlExceptionUtils.handle(SqlExceptionUtils.java:54)
	at com.clickhouse.jdbc.SqlExceptionUtils.handle(SqlExceptionUtils.java:69)
```

**Comment by qoega** [comment]
Thanks, I will look into this. It is strange that in our CI it is pretty stable.

**Comment by mrigger** [comment]
Not sure why the PR jobs are not starting. Perhaps you can close this PR and re-open it again?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -87,11 +88,17 @@ jobs:
...
         run: |
-          docker pull clickhouse/clickhouse-server:latest
-          docker run --ulimit nofile=262144:262144 --name clickhouse-server -p8123:8123 -d clickhouse/clickhouse-server:latest
+          docker pull clickhouse/clickhouse-server:head
+          docker run --ulimit nofile=262144:262144 --name clickhouse-server -p8123:8123 -d clickhouse/clickhouse-server:head
           sleep 5
       - name: Run Tests
         run: CLICKHOUSE_AVAILABLE=true mvn -Dtest=ClickHouseBinaryComparisonOperationTest,TestClickHouse,ClickHouseOperatorsVisitorTest,ClickHouseToStringVisitorTest test
+      - name: Show fatal errors
+	run: docker exec clickhouse-server grep Fatal /var/log/clickhouse-server/clickhouse-server.log
```
Comment (qoega) [inline .github/workflows/main.yml:97]:
```suggestion
      run: docker exec clickhouse-server grep Fatal /var/log/clickhouse-server/clickhouse-server.log
```

**Comment by mrigger** [comment]
Let me know when this PR is ready to be merged - I see that the CI passes now.

**Comment by qoega** [comment]
OK. I will say when, as I need to update session timeouts, but I can't reproduce locally so I will try several more runs

**Comment by qoega** [comment]
Now you can merge. I hope it will not be fail again

**Review by mrigger — APPROVED** [review]

**Pull request merged**